File: src/cli/commands/uninstall.js
import chalk from 'chalk';

export default {
    name: 'uninstall',
    explanation: `Removes a mod from the game. Usage: ${chalk.red(
        'uninstall <name>',
    )}. If you need to find the name of a mod, use 'list'`,
    matchingNames: ['u', 'uninstall'],
    execute: args => {
        console.log(chalk.red('Not available yet'));
    },
};

File: crates/cli/src/commands/uninstall.rs
use console::style;
use seahorse::{Command, Context};

/// Creates a command that uninstall a previously installed mod.
pub fn create() -> Command {
    Command::new("uninstall")
        .alias("rm")
        .usage("[name] Removes a previously installed mod. Use list to show everything that is currently installed")
        .action(handler)
}

fn handler(_: &Context) {
    println!("{}", style("Not available yet").red())
}
